frontend/src/components/organisms/MatchesList/MatchListItem.tsx

```diff
@@ -0,0 +1,20 @@
+export type MatchListItem = {
+    onClickTeam: () => void,
+    matchNumber: number | undefined,
+    level: number | undefined,
+    matchStatus: string | undefined
+    team1: {
+        readonly player1: string | undefined;
+        readonly player2: string | undefined;
+        readonly teamNumber: number | string | undefined;
+        readonly currentPlayerLevel: number | undefined;
+        readonly currentMatchNumber: number | undefined;
```

Comment:
Czym sie rozni currentMatchNumber od matchNumber wyzej?

Reply:
currentMatchNamber miał być tym numerem meczu gdzie znajduje się player w danym momencie.
Czyli np mamy mecz z poziomu 3 , a wchodzimy do innego meczu i możemy sprawdzić na którym poziomie jest dana drużyna 

- ale teraz sobie myślkę, ze można to usunąć
-

Reply:
Tez nie przesadzajmy, to chyba na razie nie potrzebne :) Najwyzej bysmy zrobili jakis inny widok odnosnie druzyn.

Reply:
To ja już stworzyłem ten inny widok odnośnie drużyn i już z niego korzystam 
 :D xD

Reply:
W sensie już w poprzednim PR

Reply:
:O Gdzie :) ? Mi chodzilo, zeby nie wszystko na jedno miejsce wrzucac, a tutaj mamy druzyny w konteksice meczu - jesli o to rozwijanie Ci chodzi?

Reply:
Ha,  jestem szybszy niż Twoje myśli!

![image](https://user-images.githubusercontent.com/41823689/114305364-2936c200-9ad8-11eb-882d-a74120858dd4.png)

Ha,  jestem szybszy niż Twoje myśli!